io.github.CatalystDevOrg.Catalyst/io.github.CatalystDevOrg.Catalyst.json

```diff
@@ -0,0 +1,64 @@
+{
+  "app-id": "io.github.CatalystDevOrg.Catalyst",
+  "base": "org.electronjs.Electron2.BaseApp",
+  "base-version": "23.08",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "run.sh",
+  "separate-locales": false,
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=wayland",
+    "--socket=x11",
```

Comment:
it either `x11` or `fallback-x11` + `wayland`. The latter mean Wayland support.

Reply:
I think it would be best to prioritize Wayland but use X11 if Wayland is unavailable.

Reply:
so do what I requested.

Reply:
Fixed with X11's default

Reply:
"--socket=fallback-x11" + "--socket=wayland" won't work. Electron does not default to wayland without a flag. You didn't pass a flag.